server/metrics/requests.go

```diff
@@ -33,18 +34,10 @@ func getRequestOkTagKeys() []string {
 		"env",
 		"db",
 		"collection",
-	}
-}
-
-func getRequestTimerTagKeys() []string {
-	return []string{
-		"grpc_method",
-		"grpc_service",
-		"tigris_tenant",
-		"grpc_service_type",
-		"env",
-		"db",
-		"collection",
+		"read_type",
```

Comment:
how are these getting used?

Reply:
These are not used yet, will be used by the query runner instrumentation, I will put these in that pr.

Reply:
I removed them for now.